src/test/java/steps/SuccessfulAccountSteps.java

```diff
@@ -0,0 +1,96 @@
+package steps;
+
+import io.cucumber.java.Before;
+import io.cucumber.java.BeforeAll;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import models.SuccessfulAccountInfo;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import pages.LoginPage;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class SuccessfulAccountSteps {
+
+   WebDriver driver = new FirefoxDriver();
+    private LoginPage loginPage = new LoginPage(driver);
+
+    @BeforeAll
+    public static void setup() {
+
+        WebDriverManager.firefoxdriver().setup();
+
+    }
+
+    @Before
+    public void the_user_is_on_dbank_homepage() {
+
+        driver.get("https://dbank-qa.wedevx.co/bank/login");
+        driver.manage().timeouts().implicitlyWait(3, TimeUnit.SECONDS);
+
+    }
+
+
+    @Given("the user logged in as {string} {string}")
+    public void the_user_logged_in_as(String username, String password) {
+        loginPage.login(username, password);
+    }
+    @When("the user creates a new checking account with the following data")
+    public void the_user_creates_a_new_checking_account_with_the_following_data(List<SuccessfulAccountInfo> successfulAccountInfoList) {
+
+        SuccessfulAccountInfo testDataForOneCheckingAccount = successfulAccountInfoList.get(0);
+
+        WebElement checkingMenu = driver.findElement(By.id("checking-menu"));
```

Comment:
Just like you did for LoginPage, it would be better to have a separate page for these locators as well